Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -170,9 +170,9 @@ Changing the return value to:0x0
  22475 ms  -[JailbreakDetectionVC isJailbroken]
 ```
 
-Please note the two calls to `-[JailbreakDetectionVC isJailbroken]`, which correspond to two physical taps on the app's GUI.
+Note the two calls to `-[JailbreakDetectionVC isJailbroken]`, which correspond to two physical taps on the app's GUI.
 
-Frida is a very powerful and versatile tool. Refer to the [Frida documentation online](https://www.frida.re/docs/home/ "Frida Documentation") for more details.
+One more way to bypass Jailbreak detection mechanism that rely on file system checks is objection. Please [find the implementation here](https://github.com/sensepost/objection/blob/master/agent/src/ios/jailbreak.ts "jailbreak.ts").
```

Comment:
You removed the "please" from the earlier sentence, but added it here? :) I would just say

> You can find the implementation in the [Jailbreak.ts file on the Objection GitHub repository](https://github.com/sensepost/objection/blob/master/agent/src/ios/jailbreak.ts "jailbreak.ts")

Reply:
detection mechanism -> detection mechanisms

Reply:
done